doc/man3/OPENSSL_ia32cap.pod

```diff
@@ -10,81 +10,77 @@ OPENSSL_ia32cap - the x86[_64] processor capabilities vector
 
 =head1 DESCRIPTION
 
-OpenSSL supports a range of x86[_64] instruction set extensions. These
-extensions are denoted by individual bits in capability vector returned
-by processor in EDX:ECX register pair after executing CPUID instruction
-with EAX=1 input value (see Intel Application Note #241618). This vector
-is copied to memory upon toolkit initialization and used to choose
-between different code paths to provide optimal performance across wide
-range of processors. For the moment of this writing following bits are
-significant:
+OpenSSL supports a range of x86[_64] instruction set extensions and
+features. These extensions are denoted by individual bits or groups of bits
+stored internally as ten 32-bit capability vectors and for simplicity
+represented logically below as five 64-bit vectors. This logical
+vector (LV) representation is used to streamline the definition of the
+OPENSSL_ia32cap environment variable.
+
+Upon toolkit initialization, the capability vectors are populated through
+successive executions of the CPUID instruction, after which any OPENSSL_ia32cap
+environment variable capability bit modifications are applied. After toolkit
+initialization is complete, populated vectors are then used to choose
+between different code paths to provide optimal performance across a wide
+range of x86[_64] based processors.
+
+Further CPUID information can be found in the Intel(R) Architecture
+Instruction Set Extensions Programming Reference, and the AMD64 Architecture
+Programmer's Manual (Volume 3).
+
+=head3 Notable Capability Bits for LV0
```

Comment:
Running `make doc-nits` gives 6 warnings, all related to `=head3`:

```
*** WARNING: =head3 without preceding higher level at line 31 in file doc/man3/OPENSSL_ia32cap.pod
*** WARNING: =head3 without preceding higher level at line 79 in file doc/man3/OPENSSL_ia32cap.pod
*** WARNING: =head3 without preceding higher level at line 116 in file doc/man3/OPENSSL_ia32cap.pod
*** WARNING: =head3 without preceding higher level at line 138 in file doc/man3/OPENSSL_ia32cap.pod
*** WARNING: =head3 without preceding higher level at line 152 in file doc/man3/OPENSSL_ia32cap.pod
*** WARNING: =head3 without preceding higher level at line 170 in file doc/man3/OPENSSL_ia32cap.pod
```

It looks like `=head2` is missing and some (or all) level 3 sections may need to be upgraded to level 2.

Reply:
Thank you for catching this, fixed!